selfdrive/car/gm/interface.py

```diff
@@ -227,6 +239,14 @@ def _update(self, c):
 
     # Enabling at a standstill with brake is allowed
     # TODO: verify 17 Volt can enable for the first time at a stop and allow for all GMs
+    #below_min_enable_speed = ret.vEgo < self.CP.minEnableSpeed or self.CS.moving_backward
```

Comment:
wont be adding this. this look like personal addons.

Reply:
ok